Source/Peripheral+Convenience.swift

```diff
@@ -79,17 +82,20 @@ extension Peripheral {
      - Returns: Observation which emits `Next` event, when specified descriptor has been found.
      Immediately after that `.Complete` is emitted.
      */
-    public func descriptor(withIdentifier identifier: DescriptorIdentifier) -> Observable<Descriptor> {
+    public func descriptor(with identifier: DescriptorIdentifier) -> Observable<Descriptor> {
         return Observable.deferred {
-            return self.characteristic(withIdentifier: identifier.characteristic)
+            return self.characteristic(with: identifier.characteristic)
                 .flatMap { characteristic -> Observable<Descriptor> in
                     if let descriptors = characteristic.descriptors,
-                        let descriptor = descriptors.findElement({ $0.UUID == identifier.UUID }) {
+                        let descriptor = descriptors.find({ $0.UUID == identifier.UUID }) {
```

Comment:
Yes. Replaced all occurrences  of `find` with `first` and also removed CollectionUtils.swift since `find` is obsolete and all wasn't used at `all`.